src/elements/styles.js

```diff
@@ -358,6 +358,69 @@ class ApplyStyles {
       this.setStyle('placeholderFocus', 'display', 'none');
     }
   }
+
+  applyBackgroundImageStyles(target) {
+    if (
```

Comment:
is this initial if statement necessary?

Reply:
i think it's fine to just check the styles dict since if mobileStyles has a background_image_url, then styles will have to have one. and if styles doesn't have one, there's a more serious bug upstream